lib/linguist/languages.yml

```diff
@@ -1138,6 +1138,16 @@ EJS:
   tm_scope: text.html.js
   ace_mode: ejs
   language_id: 95
+EML:
+  type: data
+  extensions:
+  - ".eml"
+  - ".mbox"
+  - ".mbx"
+  - ".msg"
```

Comment:
Please add a search query to your original post for **each** new extension. We need to check that they are popular enough as EML extensions on GitHub and that we won't risk miss-classifying other languages that use those extensions.

Reply:
@pchaigno done.

Reply:
Looks like `.msg` and `.mbx` have a lot of non-EML files that would be mis-classified:
- https://github.com/search?q=NOT+reply-to+extension%3Amsg
- https://github.com/search?q=NOT+from+extension%3Ambx

Where did you find these extensions?

Reply:
Again they're defined in the grammar.